sdk/cmake/CMakeMacros.cmake

```diff
@@ -466,6 +466,74 @@ function(create_iso_lists)
          INPUT ${REACTOS_BINARY_DIR}/boot/bootcdregtest.cmake.lst)
 endfunction()
 
+function(spec2def _dllname _spec_files)
+    cmake_parse_arguments(__spec2def "ADD_IMPORTLIB;NO_PRIVATE_WARNINGS;WITH_RELAY" "VERSION" "" ${ARGN})
+
+    # Get library basename
+    get_filename_component(_file ${_dllname} NAME_WE)
+
+    # Build a list of all the spec files.
+    list(LENGTH _spec_files _len)
+    if(_len GREATER 1)
```

Comment:
Is there a usecase for this?

Reply:
Not yet in current master, but soon otherwise:
https://github.com/HBelusca/reactos/commit/ff5b29e25ef05e882dd11fbc121667fb1112443e
where the list of exports is modified when the DOS support is compiled in (or excluded), by adding or not adding the dos.spec file.